docs/reference/src/main/asciidoc/configure.asciidoc

```diff
@@ -84,7 +84,7 @@ might be ignored. An example of such integrator is WildFly.
 [cols=",,",options="header",]
 |=======================================================================
 |Configuration key |Default value |Description
-|`org.jboss.weld.executor.threadPoolType` |`FIXED` |The type of the thread pool. Possible values
+|`org.jboss.weld.executor.threadPoolType` |`COMMON` |The type of the thread pool. Possible values
```

Comment:
That's not correct - `COMMON` is the default for Weld SE only. It should be something like:
> FIXED (COMMON in Weld SE)